nb/ide.launcher/netbeans.conf

```diff
@@ -57,13 +57,13 @@ netbeans_default_cachedir="${DEFAULT_CACHEDIR_ROOT}/@@metabuild.RawVersion@@"
 # here or on the command line.
 #
 # JDK 11 made GTK 3 default on Linux. Unfortunately it makes
-# NetBeans ugly on Linux with the default GTK+ Look and Feel,
+# NetBeans ugly on Linux when the non-default GTK+ Look and Feel is used,
 # as a workaround -J-Djdk.gtk.version=2.2 has been added to the 
 # default command line arguments.
 # (see: https://issues.apache.org/jira/browse/NETBEANS-1344)
 #
```

Comment:
Ah - I missed removing the comment in

https://github.com/apache/netbeans/commit/2fa8bad073ba153baae5e01359bd9f1b52eed5ce

It does not apply anymore, so it would be good if you could remove it instead of correcting it.

Reply:
i think the comment has still some value if someone uses the LAF to GTK on linux. On my system (manjaro+cinnamon), using -J-Djdk.gtk.version=2.2 still gives better results.

Not sure how it looks on ubuntu, if it isn't necessary there we can probably remove the comment.

I only changed it so that it doesn't sound as if GTK LAF is still the default on linux - we always use FlatLAF by default.

Reply:
@neilcsmith-net what do you think? should we remove the whole section or fix it in case someone is actually using GTK LAF?

Reply:
IMO, remove it.  And ideally we'd remove those legacy LAF options from the UI (while still allowing setting by CLI).

Using the GTK LAF with or without that option is a choice between really ugly and really _really_ ugly.

Reply:
roger, will remove it. There should be enough on the internet to find in case someone really, really wants to use the GTK LAF.